lib/tasks/fake_data.rake

```diff
@@ -15,19 +15,26 @@ class FakeDataGenerator
         username: user_name
     end
 
-    @stories_count.times do
+    @stories_count.times do |i|
       user = users[Random.rand(users.length-1)]
       title = Faker::Lorem.sentence(3)
       tag = Tag.find_or_create_by! tag: title.split(' ').first.downcase
-      Story.create! user: user, title: title, url: Faker::Internet.url, tags_a: [tag.tag]
+      if i.even?
+        Story.create! user: user, title: title, url: Faker::Internet.url, tags_a: [tag.tag]
+      else
+        Story.create! user: user,
+          title: title,
+          description: Faker::Lorem.paragraphs(3).join("\n\n"),
+          tags_a: [tag.tag]
+      end
     end
   end
 end
 
 desc 'Generates fake data for testing purposes'
 task fake_data: :environment do
   fail "It's not intended to be run outside development environment" unless Rails.env.development?
-  unless (User.count + Tag.count + Story.count) == 0
+  unless User.count == 1 and Tag.count == 1 and Story.count == 0
```

Comment:
I don't see a reason to change this behavior, and definitely not as part of this work.